### PR TITLE
Assets Reorganization

### PR DESCRIPTION
**Pull request by Eric-Vin**

### CODE REVIEW

File: examples/webots/city_intersection/city_intersection.scenic [hunk 1]
```diff
@@ -5,15 +5,15 @@ of two 2-lane one way roads in a city.
 
 model scenic.simulators.webots.model
 
-import shapely
 import time
 import shutil
 import os
-from pathlib import Path
+
+import shapely
```
Comment (dfremont) [inline examples/webots/city_intersection/city_intersection.scenic:12]: Added by mistake?

File: examples/webots/generic/adhoc.scenic [hunk 2]
```diff
@@ -4,11 +4,10 @@ Generate a complicated scene with ad-hoc Webots objects
 
 model scenic.simulators.webots.model
 
-import trimesh
-
 import numpy as np
 import random
-from pathlib import Path
+
+import trimesh
```
Comment (dfremont) [inline examples/webots/generic/adhoc.scenic:10]: Ditto.

File: tests/domains/driving/conftest.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 from scenic.domains.driving.roads import Network
 
-mapFolder = Path('tests')/'formats'/'opendrive'/'maps'
+mapFolder = (Path().parent.parent.parent / 'assets' / 'maps').resolve()
```
Comment (dfremont) [inline tests/domains/driving/conftest.py:11]: Hmm, this doesn't look right -- the old code seemed to assume the current directory is the root of the repository, but now we're assuming it's the folder containing this file.

File: tests/formats/opendrive/test_opendrive.py [hunk 4]
```diff
@@ -9,11 +9,8 @@
 from scenic.formats.opendrive import OpenDriveWorkspace
 from scenic.core.geometry import TriangulationError
 
-oldDir = os.getcwd()
-os.chdir(Path('tests') / 'formats' / 'opendrive')
-mapPath = Path('maps') / '**' / '*.xodr'
-maps = glob.glob(str(mapPath))
-os.chdir(oldDir)
+mapFolder = (Path().parent.parent.parent / 'assets' / 'maps').resolve()
```
Comment (dfremont) [inline tests/formats/opendrive/test_opendrive.py:12]: Same potential issue as above.

File: tests/domains/driving/conftest.py (hunk 3 above)
Comment (Eric-Vin) [reply tests/domains/driving/conftest.py]: Good catch, reverted.

File: tests/formats/opendrive/test_opendrive.py (hunk 4 above)
Comment (Eric-Vin) [reply tests/formats/opendrive/test_opendrive.py]: Good catch, reverted.

**Review by dfremont — APPROVED** [review]

**Pull request merged**